### ISSUE TITLE
Can spring team test this code base for AWS managed Kubernetes (EKS)

### ISSUE THREAD

**Issue by msidd111** [spring-cloud/spring-cloud-dataflow-server-kubernetes#297]
Now EKS is GA. Can Spring Team validate SCDF deployment for EKS. As per documentation, this kubernetes deployment is only tested against GKS, MiniKube.

**Comment by sabbyanandan** [comment]
The SCDF for Kubernetes implementation is agnostic to the Kubrnetes distribution. If anything, we test against the base Kubernetes versions in particular. You can find the compatibility matrix [here](https://github.com/spring-cloud/spring-cloud-deployer-kubernetes#kubernetes-compatibility). 

Have you tried it on AKS? We are open to any feedback that you may have.

**Comment by msidd111** [comment]
No I have not tested on AKS. We are using EKS so I need to POC SCDF on EKS..
I will retry later this week and will post my findings along with the screenshot.
Stay tune.

**Comment by msidd111** [comment]
I tried to install Spring cloud dataflow on AWS EKS without luck.
The mysql status stays pending and Scdf server status is CrashLoopBackoff. Rest of the components were installed successfully. I have used the latest image for scd server.

How to reproduce.

1. Setup AWS EKS cluster and workernodes
2. Deploy AWS Guest APP to ensure cluster is setup successfully.
3. Install Spring Cloud flow on EKS. Directions are available at URL 
https://docs.spring.io/spring-cloud-dataflow-server-kubernetes/docs/current/reference/htmlsingle/#kubernetes-getting-started
